bazel/setup_clang.sh

```diff
@@ -9,6 +9,15 @@ if [[ ! -e "${LLVM_PREFIX}/bin/llvm-config" ]]; then
   exit 1
 fi
 
+
+echo "IN FETCH"
+echo "${ENVOY_BUILD_ARCH}"
+
+if [[ "${ENVOY_BUILD_ARCH}" != "x86_64" ]]; then
+    strings /lib/aarch64-linux-gnu/libstdc++.so.6 | grep GLIBCXX
+fi
```

Comment:
I'm fine to leave these debugging lines in if they're useful, though I wonder if they will be for most developers?

Reply:
sorry i should have marked this WIP - it failed - so i retreated to envoy-build-tools to resolve

Reply:
re debugging - we probably do want something like this somewhere with a grep fail on the version - possibly this is more env detection/explicitness

having said that i added because it failed anyway - so possibly not that helpful other than not waiting to find out